cmd/util/cmd/execution-state-extract/cmd.go

```diff
@@ -422,6 +422,8 @@ func run(*cobra.Command, []string) {
 		var migs []migrations.NamedMigration
 
 		switch flagMigration {
+		case "add-keys":
```

Comment:
```suggestion
		case "add-migrationnetwork-keys":
```

maybe making the purpose a bit more clear and specific.